conans/client/tools/files.py

```diff
@@ -321,7 +321,7 @@ def collect_libs(conanfile, folder=None):
         files = os.listdir(lib_folder)
         for f in files:
             name, ext = os.path.splitext(f)
-            if ext in (".so", ".lib", ".a", ".dylib"):
+            if ext in (".so", ".lib", ".a", ".dylib", ".bc"):
```

Comment:
I'm not sure this is necessary, object files have the extension `.bc`, but Emscripten make a static library by calling the archiver like every other compiler.

Reply:
it's from https://github.com/conan-io/conan/issues/4366
@suriya1611

Reply:
Makes sense then!